src/dtos/__init__.py

```diff
@@ -0,0 +1 @@
+from .add_object_dto import AddObjectDTO
```

Comment:
Why is it added, it's good practice or why it's here?

Reply:
In general, we should use the absolute imports instead of relative imporst. Imo importing from packages increases code encapsulation